### PR TITLE
style story modal

### PR DESCRIPTION
**Pull request by shc023**
Before/after light:

Screen Shot 2020-10-15 at 10 38 35 AM
Screen Shot 2020-10-15 at 10 38 22 AM

Before/after dark:

Screen Shot 2020-10-15 at 10 38 40 AM
Screen Shot 2020-10-15 at 10 38 16 AM

### CODE REVIEW

**Review by mattvv — APPROVED** [review]
How do you make this look so easy

File: components/Comments.tsx [hunk 1]
```diff
@@ -11,6 +12,22 @@ interface CommentsParams {
   story: Story;
 }
 
+const AuthorHeader = styled.div`
+  display: flex;
+  align-items: center;
+`;
+
+const DateText = styled(Text).attrs({ small: true })`
```
Comment (mattvv) [inline components/Comments.tsx:20]: this is dope

Comment (shc023) [reply components/Comments.tsx:20]: I'm basically UX engineer (jkjk)

**Comment by shc023** [comment]
> How do you make this look so easy

I'm basically UX engineer (jkjk)

**Pull request merged**